applications/machine_learning/app_desc.rst

```diff
@@ -37,14 +37,6 @@ By default, the following sensors are used by the application:
 * nRF54L15 DK - Simulated sensor (:ref:`sensor_sim`).
   The simulated sensor generates predefined waves as acceleration.
   This development kit does not have a built-in accelerometer.
-* nRF54H20 DK - The development kit does not have a built-in accelerometer.
-  However, it supports the following configurations:
-
-   * Single-core - Accelerometer (``ADXL362``) connected to the DK with the PCA63566 shield.
-     See the shield documentation under the :file:`nrf/boards/shields/pca63566/doc/index.rst` path.
-   * Dual-core - Accelerometer (``ADXL362``) connected to the DK with the PCA63566 shield.
-     Sensor is sampled from :term:`Peripheral Processor (PPR) <Peripheral Processor (PPR, pronounced “Pepper”)>`.
-     See the shield documentation under the :file:`nrf/boards/shields/pca63566/doc/index.rst` path.
 
```

Comment:
I wouldn't remove it, as the configuration for PCA63566 is supported in the Machine Learning application. Even if the board isn't available commercially, one may design a compatible PCB with the same sensors and pin connection and it will work out of the box with the application.

Reply:
Added back.